mobileapps/views.py

```diff
@@ -1,12 +1,9 @@
 from django.contrib.auth.models import User
 from django.core.exceptions import ObjectDoesNotExist
 from django.http import Http404
-
-from rest_framework.response import Response
-from rest_framework import status
-
-from mobileapps.models import MobileApp
-from mobileapps.serializers import MobileAppSerializer
+from django.utils.translation import ugettext_lazy as _
```

Comment:
It appears import order does not conform to pep8 in this file
http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#id12